examples/archive.php

```diff
@@ -27,13 +27,14 @@
     });
 });
 
-$tar->on('error', function ($e = null) {
+$tar->on('error', function (Exception $e) {
     // should not be invoked, unless the stream is somehow interrupted
-    echo 'ERROR processing tar stream' . PHP_EOL . $e;
+    echo 'ERROR processing tar stream' . $e->getMessage() . PHP_EOL;
```

Comment:
Nit: Maybe add a colon in 32 as well? (reference line 37)

Reply:
Good catch, updated as suggested :+1: